contact.html

```diff
@@ -0,0 +1,613 @@
+                                                    itemscope="" itemtype="http://schema.org/ImageObject">
+                                                    <a href="#" class="cmp-image__link">
+                                                        <img src="./assets/yt.svg" class="cmp-image__image"
+                                                            itemprop="contentUrl" data-cmp-hook-image="image"
+                                                            alt="YouTube">
+                                                    </a>
+                                                </div>
+                                            </div>
+
+                                            <div class="image aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                                                <div data-cmp-lazythreshold="0"
+                                                    data-cmp-src="/content/leaderimpact/lang/en2/_jcr_content/content-parsys/container_428510884/container/container/container/container/image_1801542987.coreimg{.width}.svg/1669130241057/linkedin.svg"
+                                                    data-asset="/content/dam/LeaderImpact/site-./assets/linkedin.svg"
+                                                    data-title="LinkedIn" id="image-bb85cd5a39" class="cmp-image"
+                                                    itemscope="" itemtype="http://schema.org/ImageObject">
+                                                    <a href="#" class="cmp-image__link">
+                                                        <img src="./assets/linkedin.svg" class="cmp-image__image"
+                                                            itemprop="contentUrl" data-cmp-hook-image="image"
+                                                            alt="LinkedIn">
+                                                    </a>
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+          </div>
+          <!-- CONTACT -->
+          <div class="container responsivegrid contact-us aem-GridColumn aem-GridColumn--default--12">
+            <div id="contact-container" class="cmp-container">
+                <div class="aem-Grid aem-Grid--12 aem-Grid--default--12">
+
+                    <div class="container responsivegrid contact-content aem-GridColumn aem-GridColumn--default--none aem-GridColumn--default--5 aem-GridColumn--offset--default--0 aem-GridColumn--phone--none aem-GridColumn--phone--12 aem-GridColumn--offset--phone--0">
+                        <div class="cmp-container">
+                            <div class="contact-title">
```

Comment:
@Prystal254 Can't add a `<div>` inside components. Custom classes need to be on the component root element.

Reply:
done!

Reply:
@Prystal254 This is still wrong. You added the needed `.title` class on line 311 (310 now that you removed the empty line 308), but moved your custom `.contact-title` class to the child `<h1 class="cmp-title__text">`. The code for the root of this title component should be `<div class="title contact-title">`.